src/renderer/component/publishForm/view.jsx

```diff
@@ -1,884 +1,574 @@
+                  )}
+                </p>
+              )}
+              </div>
+            </section>
+
+            <section className="card card--section">
+              <div className="card__title">{__("Anonymous or under a channel?")}</div>
+              <p className="card__subtitle">
+                {__('This is a username or handle that your content can be found under.')}{' '}
+                {__('Ex. @Marvel, @TheBeatles, @BooksByJoe')}
+              </p>
+              <ChannelSection
+                channel={channel}
+                onChannelChange={this.handleChannelChange}
+              />
+            </section>
+
+            <section className="card card--section">
+              <div className="card__title">
+                {__("Where can people find this content?")}
+              </div>
+              <p className="card__subtitle">
+                {__("The LBRY URL is the exact address where people find your content (ex. lbry://myvideo).")}
+                {" "}
+                <Button fakeLink label={__('Learn more')} href="https://lbry.io/faq/naming" />
+              </p>
+              <div className="card__content">
+                <FormRow>
+                  <FormField
+                    stretch
+                    prefix={`lbry://${
+                      (channel === 'anonymous' || channel === 'new') ? '' : `${channel}/`
+                    }`}
+                    type="text"
+                    name="content_name"
+                    placeholder="myname"
+                    value={name || prefillClaim.name || ""}
+                    onChange={event => this.handleNameChange(event.target.value)}
+                    error={nameError}
+                    helper={(
+                      <BidHelpText
+                        uri={uri}
+                        isResolvingUri={isResolvingUri}
+                        winningBidForClaimUri={winningBidForClaimUri}
+                        claimIsMine={!!myClaimForUri}
+                        onEditMyClaim={this.editExistingClaim} />
+                    )}
+                    />
+                </FormRow>
+              </div>
+              <div className={classnames("card__content", { "card--disabled": !name })}>
+                <FormField
+                  className="input--price-amount"
+                  type="number"
+                  name="content_bid"
+                  step="any"
+                  label={__('Deposit')}
+                  postfix="LBC"
+                  value={bid}
+                  error={bidError}
+                  min="0"
+                  disabled={!name}
+                  onChange={event => this.handleBidChange(event.target.value)}
+                  helper={__('This LBC remains yours and the deposit can be undone at any time.')}
+                  placeholder={winningBidForClaimUri ? winningBidForClaimUri + 10 : 10}
```

Comment:
I thought we shifted from 10 as the placeholder to a smaller amount, like 0.01.

I know some people were confused that placeholders were default or suggested values.

And even if they're not confused, the value here will act as a psychological anchor. We should consider where we want to anchor people.